src/components/Pagination/index.jsx

```diff
@@ -8,25 +8,31 @@ const Pagination = ({
   updatePageIndex,
   pageIndex,
   pageInterval,
+  pagesToDisplay,
   numRules,
 }) => {
-  const endPage = pageIndex + (5 - (pageIndex % 5 === 0 ? 5 : pageIndex % 5));
-  const startPage = endPage - 4;
+  const endPage =
+    pageIndex +
+    (pagesToDisplay -
+      (pageIndex % pagesToDisplay === 0
+        ? pagesToDisplay
+        : pageIndex % pagesToDisplay));
+  const startPage = endPage - (pagesToDisplay - 1);
+  const totalPages = Math.ceil(numRules / pageInterval);
   return (
     <div className="pagination pagination-container">
       <a
-        className={
-          `btn btn-secondary btn-small pagination-prev ${ 
-          pageIndex <= 5 ? "is-disabled" : ""}`
-        }
+        className={`btn btn-secondary btn-small pagination-prev ${
+          pageIndex <= pagesToDisplay ? "is-disabled" : ""
+        }`}
         onClick={prevPageIndex}
-        onKeyDown={() => prevPageIndex}
+        onKeyDown={prevPageIndex}
         role="button"
         tabIndex="0"
       >
         Prev
       </a>
-      {pageIndex > 5 && (
+      {pageIndex > pagesToDisplay && (
```

Comment:
can you store `pageIndex > pagesToDisplay` in a descriptive variable so we know what that code is checking for? so something like `shouldDisplayPaginationPage1` or whatever name you come up with